demo/site/next.config.mjs

```diff
@@ -29,6 +29,16 @@ const nextConfig = {
     },
     cacheHandler: process.env.REDIS_ENABLED === "true" ? import.meta.resolve("./dist/cache-handler.js").replace("file://", "") : undefined,
     cacheMaxMemorySize: process.env.REDIS_ENABLED === "true" ? 0 : undefined, // disable default in-memory caching
+    rewrites: () => {
+        return {
+            afterFiles: [
+                {
+                    source: "/:path(_next|assets)/:file*",
```

Comment:
```suggestion
                    // Show a 404 instead of trying to render page for paths starting with /_next/ or /assets/ as they don't get rewritten in DomainRewriteMiddleware and cause errors in ...path page
                    source: "/:path(_next|assets)/:file*",
```